src/System.Runtime.Extensions/src/System/ApplicationId.cs

```diff
@@ -41,7 +41,8 @@ public ApplicationId(byte[] publicKeyToken, string name, Version version, string
 
         public override string ToString ()
         {
-            StringBuilder sb = StringBuilderCache.Acquire();
+            Span<char> charSpan = stackalloc char[127];
```

Comment:
127 is a weird number. Why not use 128 which spans from 0 - 127?

Reply:
The space on the stack is aligned to pointer sized chunks anyway. If you use 127, there will be one unused byte.

128 is better.

Reply:
fixed